apps/fyle/helpers.py

```
@@ -324,10 +325,21 @@ def check_interval_and_sync_dimension(workspace_id: int):
     workspace = Workspace.objects.get(id=workspace_id)
+    last_export_detail = LastExportDetail.objects.get(workspace_id=workspace_id)
+    configuration = Configuration.objects.get(workspace_id=workspace_id)
     fyle_credentials = FyleCredential.objects.get(workspace_id=workspace_id)
 
     platform = PlatformConnector(fyle_credentials)
     platform.import_fyle_dimensions(is_export=is_export)
+    unmapped_card_count = ExpenseAttribute.objects.filter(
+        attribute_type="CORPORATE_CARD", workspace_id=workspace_id, active=True, mapping__isnull=True
+    ).count()
+    if unmapped_card_count != last_export_detail.unmapped_card_count:
+        if configuration.corporate_credit_card_expenses_object == 'CREDIT CARD CHARGE':
```

Ruff (0.12.2)

Use a single `if` statement instead of nested `if` statements

(SIM102)